galaxy.yml

```
@@ -2,7 +2,7 @@ namespace: stackhpc
 name: hashicorp
 description: >
   Hashicorp Vault/Consul deployment and configuration
-version: "2.6.0"
+version: "2.7.0"
```

```suggestion
version: "2.6.1"
```
Would consider this more of a bugfix. What do you think?